actuator/src/main/java/org/tron/core/vm/PrecompiledContracts.java

```diff
@@ -983,11 +983,13 @@ public Pair<Boolean, byte[]> execute(byte[] rawData) {
   public static class BatchValidateSign extends PrecompiledContract {
 
     private static final ExecutorService workers;
+    private static final String workersName = "validate-sign-contract";
```

Comment:
Should this managed by ExecutorServiceManager?

Reply:
Yes, do you have any good ideas?

Reply:
So we can just move the excutors name and the construction into ExecutorServiceManager, then the lifecycle and all the executors is controlled in ExecutorServiceManager , a certain executor shall be obtained by using a name(may be obtained by spring injection).

Reply:
Thread pool shutdown is better controlled by the service, not the manager but this one is a special case, so maybe it can be done this way.